task_handler_layer/task_buoy/src/buoy_detection.cpp

```diff
@@ -36,7 +36,7 @@ void callback(task_buoy::buoyConfig &config, uint32_t level)
   t2max = config.t2max_param;
   t3min = config.t3min_param;
   t3max = config.t3max_param;
-  ROS_INFO("Reconfigure Request : New parameters : %d %d %d %d %d %d", t1min, t1max, t2min, t2max, t3min, t3max);
+  ROS_INFO("Buoy_Reconfigure Request:New params : %d %d %d %d %d %d", t1min, t1max, t2min, t2max, t3min, t3max);
```

Comment:
You see if i add ros::thisnode then the line length gives roslint error and i can't use format.cpp in ip nodes. And it's just here that i have hardcoded, now here else.